app/team.go

```diff
@@ -1127,6 +1127,45 @@ func (a *App) prepareInviteNewUsersToTeam(teamId, senderId string) (*model.User,
 	return user, team, nil
 }
 
+func genEmailInviteWithErrorList(emailList []string) []*model.EmailInviteWithError {
+	var invitesNotSent []*model.EmailInviteWithError
+	for i := range emailList {
+		invite := &model.EmailInviteWithError{
+			Email: emailList[i],
+			Error: model.NewAppError("inviteUsersToTeam", "api.team.invite_members.limit_reached.app_error", map[string]interface{}{"Addresses": emailList[i]}, "", http.StatusBadRequest),
+		}
+		invitesNotSent = append(invitesNotSent, invite)
```

Comment:
nit: if the previous optimization is applied, this could be `invitesNotSent[i] = invite`